modules/dxp/apps/analytics/analytics-reports-web/test/js/components/BasicInformation.js

```diff
@@ -55,7 +55,8 @@ describe('BasicInformation', () => {
 
 		expect(getByText(testProps.canonicalURL)).toBeInTheDocument();
 
-		const formattedPublishDate = 'September 20, 2021';
+		const formattedPublishDate = '20 September 2021';
```

Comment:
Hmm, this is a bit of a smell that we have to update this test. How come this is changed?

Reply:
I think we should change our mock to fit the existing tests, rather than changing the tests

Reply:
Alright, I think that this one is failing on master and I assumed it started failing because of my changes.